### ISSUE TITLE
NPE in HaxeExpressionEvaluator.

### ISSUE THREAD

**Issue by EricBishton** [HaxeFoundation/intellij-haxe#954]
Randomly ran into this while editing.  Not sure how to repro.

```
In file: file:///home/sni/sandbox/uwongke/poptropica-openfl/src/game/creators/scene/RaceSegmentCreator.hx

java.lang.NullPointerException
	at com.intellij.plugins.haxe.model.type.HaxeExpressionEvaluator.handle(HaxeExpressionEvaluator.java:81)
	at com.intellij.plugins.haxe.model.type.HaxeExpressionEvaluator._handle(HaxeExpressionEvaluator.java:207)
	at com.intellij.plugins.haxe.model.type.HaxeExpressionEvaluator.handle(HaxeExpressionEvaluator.java:68)
	at com.intellij.plugins.haxe.model.type.HaxeExpressionEvaluator.evaluate(HaxeExpressionEvaluator.java:59)
	at com.intellij.plugins.haxe.model.type.HaxeTypeResolver.evaluateFunction(HaxeTypeResolver.java:515)
	at com.intellij.plugins.haxe.model.type.HaxeTypeResolver.getPsiElementType(HaxeTypeResolver.java:492)
	at com.intellij.plugins.haxe.model.type.HaxeTypeResolver.getPsiElementType(HaxeTypeResolver.java:443)
	at com.intellij.plugins.haxe.ide.annotator.AssignExpressionChecker.check(HaxeSemanticAnnotator.java:923)
	at com.intellij.plugins.haxe.ide.annotator.HaxeSemanticAnnotator.analyzeSingle(HaxeSemanticAnnotator.java:82)
	at com.intellij.plugins.haxe.ide.annotator.HaxeSemanticAnnotator.annotate(HaxeSemanticAnnotator.java:61)
	at com.intellij.codeInsight.daemon.impl.DefaultHighlightVisitor.runAnnotators(DefaultHighlightVisitor.java:121)
	at com.intellij.codeInsight.daemon.impl.DefaultHighlightVisitor.visit(DefaultHighlightVisitor.java:86)
	at com.intellij.codeInsight.daemon.impl.GeneralHighlightingPass.runVisitors(GeneralHighlightingPass.java:351)
	at com.intellij.codeInsight.daemon.impl.GeneralHighlightingPass.lambda$collectHighlights$5(GeneralHighlightingPass.java:284)
	at com.intellij.codeInsight.daemon.impl.GeneralHighlightingPass.analyzeByVisitors(GeneralHighlightingPass.java:311)
	at com.intellij.codeInsight.daemon.impl.GeneralHighlightingPass.lambda$analyzeByVisitors$6(GeneralHighlightingPass.java:314)
	at com.intellij.codeInsight.daemon.impl.DefaultHighlightVisitor.analyze(DefaultHighlightVisitor.java:70)
	at com.intellij.codeInsight.daemon.impl.GeneralHighlightingPass.analyzeByVisitors(GeneralHighlightingPass.java:314)
	at com.intellij.codeInsight.daemon.impl.GeneralHighlightingPass.collectHighlights(GeneralHighlightingPass.java:281)
	at com.intellij.codeInsight.daemon.impl.GeneralHighlightingPass.collectInformationWithProgress(GeneralHighlightingPass.java:225)
	at com.intellij.codeInsight.daemon.impl.ProgressableTextEditorHighlightingPass.doCollectInformation(ProgressableTextEditorHighlightingPass.java:84)
	at com.intellij.codeHighlighting.TextEditorHighlightingPass.collectInformation(TextEditorHighlightingPass.java:55)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$null$1(PassExecutorService.java:429)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1106)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$doRun$2(PassExecutorService.java:422)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:591)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:537)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:59)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.doRun(PassExecutorService.java:421)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$run$0(PassExecutorService.java:397)
	at com.intellij.openapi.application.impl.ReadMostlyRWLock.executeByImpatientReader(ReadMostlyRWLock.java:164)
	at com.intellij.openapi.application.impl.ApplicationImpl.executeByImpatientReader(ApplicationImpl.java:204)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.run(PassExecutorService.java:395)
	at com.intellij.concurrency.JobLauncherImpl$VoidForkJoinTask$1.exec(JobLauncherImpl.java:161)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1020)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1656)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1594)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:177)
```

**Comment by EricBishton** [comment]
It's actually throwing the NPE while trying to log an error message. 
 bfc47e79d0e36ff562cafad2ee7d8aad7b3244ac should fix it.

**Comment by EricBishton** [comment]
Fixed in #959
